### ISSUE TITLE
highlightOnTouch does not work if elevation is set

### ISSUE THREAD

**Issue by maxapps** [eclipsesource/tabris-js#744]
Setting the new (Tabris 1.5) elevation property on a Composite widget seems to disable the highlightOnTouch setting.

```javascript
{elevation: 0, highlightOnTouch: true}  // works as expected
{elevation: 1, highlightOnTouch: true}  // no highlight when tapped
```

**Comment by mpost** [comment]
You are right about your assumption. We are looking into that...

Speaking of `highlightOnTouch`... Starting with Tabris.js 1.6 you will see a ripple effect for `highlightOnTouch` on all Android 6+ devices.

**Comment by maxapps** [comment]
Fantastic news. I'm toying with making a material JS component library and both features will be required.

**Comment by mpost** [comment]
Fixed for the upcoming 1.6 release.
